modules/swagger-codegen/src/main/resources/JavaSpring/libraries/spring-boot/pom.mustache

```diff
@@ -44,6 +44,10 @@
             <artifactId>spring-boot-starter-tomcat</artifactId>
             <scope>provided</scope>
         </dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
```

Comment:
It's less intrusive to let the user add spring-boot-starter-data-jpa if he needs JPA and it will not cause any problem to have spring-data-commons in parallel.